symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -42,6 +42,7 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
             $loader->load($confDir.'/packages/'.$this->environment.'/**/*'.self::CONFIG_EXTS, 'glob');
         }
         $loader->load($confDir.'/services'.self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir.'/services_'.$this->environment.self::CONFIG_EXTS, 'glob');
```

Comment:
What if puts the environment directories at root instead? it'll keep the same convention just to app-conf.
```
├── config
│   ├── test/
│   │   └── services.yaml
│   ├── packages/
│   │   └── dev/
│   └── services.yaml
```
```php
        if (is_dir($confDir.'/'.$this->environment)) {
            $loader->load($confDir.'/'.$this->environment.'/**/*'.self::CONFIG_EXTS, 'glob');
        }
```

Reply:
See symfony/flex#141 too, I think 😜